content/02.body.md

```diff
@@ -1,13 +1,44 @@
 ## Introduction
 
-### Node degree
-
-![Degree figure](){#fig:degree width="100%"}
-
-### Edge prediction
-
-### Feature-degree correlation
-
+Most biomedical data networks are imperfect representations of the true set of relationships.
+Real networks often mistakenly include edges that do not exist and exclude edges that do exist.
+How well a network represents the true relationships it attempts to represent depends on a number of factors, especially the methods used to generate the data in the network [@doi:10.1016/j.jprot.2014.01.020; @doi:10.3389/fgene.2015.00260; @doi:10.1038/nbt1116].
+We define "degree bias" as the type of misrepresentation that occurs when the fraction of incorrectly existent/nonexistent relationships depends on the number of connections that nodes make (their "degrees").
+Inspection bias indicates that entities are not uniformly studied [@doi:10.1038/nature04209], and it is likely to cause degree bias when networks are constructed using hypothesis-driven findings extracted from the literature, as newly-discovered relationships are not randomly sampled from the set of all true relationships.
+
+For example, the number of publications mentioning a gene has little correlation with its degree in a systematically-derived protein interaction network [Figure 6A in @doi:10.1016/j.cell.2014.10.050].
+Therefore, the high correlation between number of publications and degree in low-throughput interaction networks is almost entirely the result of inspection bias.
+This evidence suggests that many poorly studied genes have similar numbers of interactions as the genes scientists have preferentially examined.
+
+Depending on the type of data being represented, degree biases can also arise due to experimental methods or other factors [@doi:10.1016/j.jprot.2014.01.020].
```

Comment:
This feels like maybe it belongs immediately after the sentence defining degree bias.